Client/mods/deathmatch/logic/CPacketHandler.cpp

```diff
@@ -452,6 +457,9 @@ void CPacketHandler::Packet_ServerJoined(NetBitStreamInterface& bitStream)
     g_pClientGame->m_pLocalPlayer->CallEvent("onClientPlayerJoin", Arguments, true);
 
     g_pCore->UpdateRecentlyPlayed();
+
+    if (g_pCore->GetNetwork()->GetServerBitStreamVersion() < 0x06D)
+        g_pCore->GetDiscordManager()->SetState(SString("%i Players", ucNumberOfPlayers), [](EDiscordRes) {});
```

Comment:
This needs to be updated too (I meant the duplication of SetState between here, [`Packet_ServerDisconnected`](https://github.com/multitheftauto/mtasa-blue/pull/1031/files#diff-3ed6112654809f7609a791c26259f645R1004) and [`CServerInfo`](https://github.com/multitheftauto/mtasa-blue/pull/1031/files#diff-be4af7a17233d412f46d4e3cecb371feR22))